src/languageService/kustoLanguageService.ts

```diff
@@ -1196,7 +1198,8 @@ class KustoLanguageService implements LanguageService {
             return Promise.as(undefined);
         }
 
-        return Promise.as({ contents: quickInfo.Text });
+        // \n\n doesn't work in a markdown, it won't create an empty line. So instead use "\n\n&nbsp;\n\n".
+        return Promise.as({ contents: quickInfo.Text.replace("\n\n", "\n\n&nbsp;\n\n") });
```

Comment:
can this be null or undefined? if so we can crash here

Reply:
The previous line is `if (!quickInfo || !quickInfo.Text) {`